### PR TITLE
Verified input integration test

### PR DESCRIPTION
**Pull request by johnmccutchan**
This is an integration test associated with https://github.com/flutter/engine/pull/49268

### CODE REVIEW

File: dev/integration_tests/android_verified_input/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_driver:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  test: 1.24.9
```
Comment (dnfield) [inline dev/integration_tests/android_verified_input/pubspec.yaml:23]: nit: newline at eof

**Review by dnfield — APPROVED** [review]
LGTM

File: dev/integration_tests/android_verified_input/pubspec.yaml (hunk 1 above)
Comment (johnmccutchan) [reply dev/integration_tests/android_verified_input/pubspec.yaml:23]: Done.

**Comment by christopherfujino** [comment]
FYI elliette just pinned path_provider_android in https://github.com/flutter/flutter/pull/140810/files because of https://github.com/flutter/flutter/issues/140796

**Comment by johnmccutchan** [comment]
> FYI elliette just pinned path_provider_android in https://github.com/flutter/flutter/pull/140810/files because of #140796

What should I do with this information?

**Pull request merged**